### PR TITLE
feat(integration-templates): improve google-drive templates

### PR DESCRIPTION
**Pull request by hassan254-prog**
## Describe your changes

- Improve `google-drive` integration templates
- Follow up of [2665](https://github.com/NangoHQ/nango/pull/2665)

## Issue ticket number and link
[EXT-145](https://linear.app/nango/issue/EXT-145/refractor-google-drive-integration)

### CODE REVIEW

**Review by bodinsamuel — APPROVED** [review]
Looks good. I'm wondering what happen in the UI if you have fetch-pdf enabled

**Comment by hassan254-prog** [comment]
@bodinsamuel 
image

File: integration-templates/google-drive/types.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+interface MimeTypeMapping {
+    mimeType: string;
+    responseType: 'text' | 'stream';
+}
+
+export const mimeTypeMapping: Record<string, MimeTypeMapping> = {
+    // Documents
+    'application/vnd.google-apps.document': { mimeType: 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', responseType: 'text' },
```
Comment (TBonnin) [inline integration-templates/google-drive/types.ts:20]: where does this mapping and the one below come from?

**Review by TBonnin — APPROVED** [review]

File: integration-templates/google-drive/types.ts (hunk 1 above)
Comment (hassan254-prog) [reply integration-templates/google-drive/types.ts:20]: - `application/vnd.google-apps.document`: This is a MIME type used by Google Drive to represent Google Docs documents.
- `mimeType`: `application/vnd.openxmlformats-officedocument.wordprocessingml.document` specifies the desired MIME type for the converted file, which is the standard MIME type for Microsoft Word documents in the Office Open XML format.

**Review by khaliqgant — APPROVED** [review]

**Pull request merged**